libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -137,7 +137,13 @@ SymbolicFunctionVariable::SymbolicFunctionVariable(
 	EncodingContext& _context
 ):
 	SymbolicVariable(_type, _type, move(_uniqueName), _context),
-	m_declaration(m_context.newVariable(currentName(), m_sort))
+	m_declaration(m_context.newVariable(currentName(), m_sort)),
+	m_abstract(
```

Comment:
It's annoying that this is now repeated, but having a separate private constructor only for this would force that constructor to also call `SymbolicVariable`, so we'd actually need 2. Not really sure what the best way here is.

Reply:
Why would the private c'tor also need to call `SymbolicVariable`?

Reply:
Member-initialization (see below)?

Reply:
I don't get it. I mean you can still call `SymbolicVariable` here and call the (theoretical) private c'tor from here, too?

Reply:
Yes, that works for this constructor, but compilation fails for the "new" constructor because it needs to call the parent constructor.